packages/dtslint/src/checks.ts

```diff
@@ -56,16 +56,24 @@ export async function checkPackageJson(dirPath: string, typesVersions: readonly
   }
 }
 
+/**
+ * numbers in `CompilerOptions` might be enum values mapped from strings
```

Comment:
which specific properties have this? I assume it's "module".

Reply:
`module` is the only one that matters here, but there are other enums that aren’t relevant to DT. Most numbers in `CompilerOptions` are actually enums represented by strings in the raw form. The only real number I can think of is `maxNodeModulesJsDepth`.